modules/installation-obtaining-installer.adoc

```diff
@@ -90,8 +108,17 @@ endif::restricted[]
 
 .Prerequisites
 
-ifdef::ibm-z,ibm-z-kvm,private[* You have a machine that runs Linux, for example Red Hat Enterprise Linux 8, with 500 MB of local disk space.]
-ifndef::ibm-z,ibm-z-kvm,private[* You have a computer that runs Linux or macOS, with 500 MB of local disk space.]
+ifdef::ibm-z,ibm-z-kvm,private,vsphere,vmc[]
+* You have a machine that runs Linux, for example Red Hat Enterprise Linux 8, with 500 MB of local disk space.
+endif::ibm-z,ibm-z-kvm,private,vsphere,vmc[]
+ifdef::vsphere,vmc[]
++
+[IMPORTANT]
+====
+A known issue exists when you attempt to run the `openshift-installer` on macOS. The issue specifically relates to the `golang` compiler. For more information about this issue, see the section named "Known Issues" in the _{product-title} {product-version} release notes_ document.
```

Comment:
I understand not putting the whole known issue into this note, but I am curious if you could mention the ultimate impact of the known issue here (that the installation fails). I think it can be tightened up a bit, too. 

Suggestion to take or leave: "If you attempt to run the `openshift-installer` on macOS, a known issue related to the `golang` compiler causes the installation to fail. For more information..."

Reply:
Thanks, @ousleyp . Great suggested. I happily implemented it :) . I'll add a merge label to this PR.